tests/system/solvers/test_45_FilterSolutions/test_filter/test_attributes.py

```diff
@@ -373,51 +641,119 @@ def test_asymmetric_high_order(self, lumped_design):
+        with pytest.raises(RuntimeError) as info:
+            lumped_design.attributes.asymmetric_low_stop_band_attenuation_db = "40"
+        if config["desktopVersion"] > "2025.1":
+            assert info.value.args[0] == "The Butterworth filter does not have stop band attenuation"
+        else:
+            assert info.value.args[0] == "The Butterworth filter does not support asymmetric option"
         lumped_design.attributes.filter_type = FilterType.ELLIPTIC
+        with pytest.raises(RuntimeError) as info:
+            lumped_design.attributes.asymmetric_low_stop_band_attenuation_db = "40"
+        if config["desktopVersion"] > "2025.1":
+            assert info.value.args[0] == "The asymmetric option is disabled"
+        else:
+            assert info.value.args[0] == "The Elliptic filter does not support asymmetric option"
         lumped_design.attributes.asymmetric = True
         assert lumped_design.attributes.asymmetric_low_stop_band_attenuation_db == "60"
         lumped_design.attributes.asymmetric_low_stop_band_attenuation_db = "40"
         assert lumped_design.attributes.asymmetric_low_stop_band_attenuation_db == "40"
 
     def test_asymmetric_high_stop_band_attenuation_db(self, lumped_design):
         lumped_design.attributes.filter_class = FilterClass.BAND_PASS
+        with pytest.raises(RuntimeError) as info:
+            lumped_design.attributes.asymmetric_high_stop_band_attenuation_db = "40"
+        if config["desktopVersion"] > "2025.1":
+            assert info.value.args[0] == "The Butterworth filter does not have stop band attenuation"
+        else:
+            assert info.value.args[0] == "The Butterworth filter does not support asymmetric option"
         lumped_design.attributes.filter_type = FilterType.ELLIPTIC
+        with pytest.raises(RuntimeError) as info:
+            lumped_design.attributes.asymmetric_high_stop_band_attenuation_db = "40"
+        if config["desktopVersion"] > "2025.1":
+            assert info.value.args[0] == "The asymmetric option is disabled"
+        else:
+            assert info.value.args[0] == "The Elliptic filter does not support asymmetric option"
         lumped_design.attributes.asymmetric = True
         assert lumped_design.attributes.asymmetric_high_stop_band_attenuation_db == "60"
         lumped_design.attributes.asymmetric_high_stop_band_attenuation_db = "40"
         assert lumped_design.attributes.asymmetric_high_stop_band_attenuation_db == "40"
 
     def test_gaussian_transition(self, lumped_design):
+        with pytest.raises(RuntimeError) as info:
+            lumped_design.attributes.gaussian_transition = GaussianTransition.TRANSITION_3_DB
+        if config["desktopVersion"] > "2025.1":
+            assert info.value.args[0] == "The Butterworth filter does not have Gaussian transition"
+        else:
+            assert info.value.args[0] == "The Butterworth filter has no tranisiotn parameter option"
```

Comment:
```suggestion
            assert info.value.args[0] == "The Butterworth filter has no transition parameter option"
```

Have these tests been run with 25R1? If yes and this test passed, we need to update the message in the DLL as well.

Reply:
The message is mistyped in 25R1. to pass the test with 25R1 this condition is added.